### ISSUE TITLE
When running `skunk -b master` I get a ENOENT error

### ISSUE THREAD

**Issue by etagwerker** [fastruby/skunk#12]
I just tried to run `skunk -b master` using skunk v0.3.0 and I found this issue:

```
Traceback (most recent call last):
	11: from /Users/etagwerker/.rvm/gems/ruby-2.6.3/bin/ruby_executable_hooks:24:in `<main>'
	10: from /Users/etagwerker/.rvm/gems/ruby-2.6.3/bin/ruby_executable_hooks:24:in `eval'
	 9: from /Users/etagwerker/.rvm/gems/ruby-2.6.3/bin/skunk:23:in `<main>'
	 8: from /Users/etagwerker/.rvm/gems/ruby-2.6.3/bin/skunk:23:in `load'
	 7: from /Users/etagwerker/.rvm/gems/ruby-2.6.3/gems/skunk-0.3.0/exe/skunk:10:in `<top (required)>'
	 6: from /Users/etagwerker/.rvm/gems/ruby-2.6.3/gems/skunk-0.3.0/lib/skunk/cli/application.rb:16:in `execute'
	 5: from /Users/etagwerker/Projects/fastruby/rubycritic/lib/rubycritic/commands/compare.rb:20:in `execute'
	 4: from /Users/etagwerker/Projects/fastruby/rubycritic/lib/rubycritic/commands/compare.rb:38:in `compare_branches'
	 3: from /Users/etagwerker/Projects/fastruby/rubycritic/lib/rubycritic/commands/compare.rb:65:in `compare_code_quality'
	 2: from /Users/etagwerker/.rvm/gems/ruby-2.6.3/gems/skunk-0.3.0/lib/skunk/cli/commands/compare.rb:35:in `build_details'
	 1: from /Users/etagwerker/.rvm/gems/ruby-2.6.3/gems/skunk-0.3.0/lib/skunk/cli/commands/compare.rb:35:in `open'
/Users/etagwerker/.rvm/gems/ruby-2.6.3/gems/skunk-0.3.0/lib/skunk/cli/commands/compare.rb:35:in `initialize': No such file or directory @ rb_sysopen - /Users/etagwerker/Projects/fastruby/rubycritic/tmp/rubycritic/compare/build_details.txt (Errno::ENOENT)
```

It seems that it is expecting a `compare` directory to be there but it is not.

A workaround to solve this problem is to do this:

```
$ mkdir tmp/rubycritic/compare
```

Then `skunk -b master` seems to work fine.

**Comment by vnhnhm** [comment]
The error is still present in version 0.3.2 (last till now). What about a quick check if the path exists, if not then create it?

```ruby
Dir.mkdir(path) unless File.exists?(path)
```

Or to handle the creation of the file with a command flag when the file is really needed?

**Comment by etagwerker** [comment]
@vnhnhm I think that is a great idea. Do you want to take a stab at it? 

PS: Sorry for the delay. I was on vacation.

**Comment by etagwerker** [comment]
Closed by #23
